=0.15

```diff
@@ -0,0 +1,6 @@
+Requirement already satisfied: pyopenssl in ./venv/lib/python3.6/site-packages (19.0.0)
```

Comment:
I'm not sure this file should be here!

Reply:
Ah, damn. I thought I deleted that.

Anyway, it's deleted now.